gnd/src/main/res/layout/offline_areas_frag.xml

```diff
@@ -20,56 +20,51 @@
   xmlns:app="http://schemas.android.com/apk/res-auto">
 
   <data>
-
     <variable
       name="viewModel"
       type="com.google.android.gnd.ui.offlinearea.OfflineAreasViewModel" />
   </data>
 
-  <LinearLayout
+  <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:orientation="vertical">
+    android:fitsSystemWindows="true">
```

Comment:
Not a major issue, esp. if it saves us code complexity, but the system inset areas shows as grey instead of blue in Offline areas now:
![image](https://user-images.githubusercontent.com/228050/82760079-534b9c80-9dbf-11ea-865e-18019c6bd474.png)

Before:
![image](https://user-images.githubusercontent.com/228050/82760108-737b5b80-9dbf-11ea-9609-67f82927ae65.png)

Ideal would be as in the expanded feature sheet:
![image](https://user-images.githubusercontent.com/228050/82760118-82620e00-9dbf-11ea-8068-b815ad4d4f77.png)

If it's not easy to fix, this is really a P3 UI tweak that can be punted.

Reply:
It was like that because of hardcoded padding. After fixing that, it is showing the same status bar scrim as the home screen. 

To fix it system-wide, I am thinking about using colors (with alpha value) in the status_bar_scrim of MainActivity. It would be the dark shade of the toolbar. (In a separate PR)

Reply:
Great, I think that's sufficient. Thanks for the fix!